samples/appendingEvents.go

```diff
@@ -0,0 +1,142 @@
+package samples
+
+import (
+	"context"
+	"log"
+
+	"github.com/EventStore/EventStore-Client-Go/client"
+	"github.com/EventStore/EventStore-Client-Go/messages"
+	"github.com/EventStore/EventStore-Client-Go/options"
+	"github.com/EventStore/EventStore-Client-Go/stream_position"
+	"github.com/EventStore/EventStore-Client-Go/streamrevision"
+)
+
+type TestEvent struct {
+	id            string
```

Comment:
Will it Marshal with JSON? It's late and I could be wrong but if fields are not public it's likely will be `{}`.

If I'm right I guess that it may be nice to have some tests for that package to ensure that examples are working as expected.